platform/src/main/java/org/hillview/main/BatchLogAnalysis.java

```diff
@@ -0,0 +1,188 @@
+package org.hillview.main;
+
+import org.hillview.dataset.LocalDataSet;
+import org.hillview.dataset.api.Empty;
+import org.hillview.dataset.api.IDataSet;
+import org.hillview.dataset.api.IMap;
+import org.hillview.maps.ExtractValueFromKeyMap;
+import org.hillview.maps.FindFilesMapper;
+import org.hillview.maps.LoadFilesMapper;
+import org.hillview.sketches.*;
+import org.hillview.storage.FileSetDescription;
+import org.hillview.storage.IFileReference;
+import org.hillview.table.api.ITable;
+import org.hillview.utils.Converters;
+import org.hillview.utils.JsonList;
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.cli.*;
+
+
+public class BatchLogAnalysis {
```

Comment:
You should add a comment describing what this class is supposed to do.

Reply:
The comments should document the expected input structure: logs that are in the syslog rfc5424 format, and that have a field called "errorCode" in the "structured data" column.